Sources/StreamChatUI/Appearance+Images.swift

```diff
@@ -189,6 +189,14 @@ public extension Appearance {
 
         public var messageActionInlineReply: UIImage = loadImageSafely(with: "icn_inline_reply")
         public var messageActionThreadReply: UIImage = loadImageSafely(with: "icn_thread_reply")
+        public var messageActionMarkUnread: UIImage = {
+            if #available(iOS 13.0, *) {
+                return UIImage(systemName: "message.badge")!
```

Comment:
can't we fallback to the `loadImageSafely` here as well, instead of force unwrapping?